GNUmakefile

```diff
@@ -68,6 +68,11 @@ tpgtools:
 	cd tpgtools;\
 		go run . --output $(OUTPUT_PATH) --version $(VERSION) $(tpgtools_compile)
 
+test:
+	cd mmv1;\
+		bundle; \
+		bundle exec rake test
```

Comment:
Adding this make target was necessary to run the non-e2e tests locally as per the developer guide:

> Run unit tests
> Unit tests (that is, tests that do not interact with the GCP API) are very fast and you can generally run them all if you have changed any of them:
> 
> make test

https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/run-provider-tests.md#run-unit-tests

Reply:
This documentation is specific to the downstream providers.. Lets omit this